src/scripts/upgrades/Upgrade.ts

```diff
@@ -0,0 +1,76 @@
+///<reference path="../GameConstants.ts"/>
+///<reference path="Cost.ts"/>
+///<reference path="CostFactory.ts"/>
+
+class Upgrade {
+    name: any;
+    displayName: string;
+    maxLevel: number;
+    _level: KnockoutObservable<number> = ko.observable();
+    increasing: boolean;
```

Comment:
Might be worth adding a comment here for when we forget what this is.

Reply:
Done